app/script/view_model/VideoCallingViewModel.js

```diff
@@ -53,14 +53,10 @@ z.viewModel.VideoCallingViewModel = class VideoCallingViewModel {
     this.currentDeviceId = this.mediaRepository.devices_handler.current_device_id;
     this.currentDeviceIndex = this.mediaRepository.devices_handler.current_device_index;
 
-    this.localVideoStream = this.mediaRepository.stream_handler.localMediaStream;
+    const streamHandler = this.mediaRepository.stream_handler;
+    this.localVideoStream = streamHandler.localMediaStream;
     this.remoteVideoStreams = ko.pureComputed(() => {
-      // FIXME the media repository should release the memory of inactive video streams
-      // Right now, only the status is changed but we keep the stream in memory
-      // until the next page reload.
-      let videoStreams = this.mediaRepository.stream_handler.remote_media_streams
-        .video()
-        .filter(stream => stream.active);
+      let videoStreams = streamHandler.remote_media_streams.activeVideo();
 
       if (videoStreams.length > 1) {
         videoStreams = videoStreams.concat(this.localVideoStream());
```

Comment:
Woups, this should be removed I believe. 
We are doing this twice (in the grid and in the callingviewmodel).